health/notifications/alarm-notify.sh.in

```diff
@@ -2669,10 +2654,10 @@ if [ -n "$total_warn_alarms" ]; then
    while read -d, -r pair; do
        IFS='=' read -r key val <<<"$pair"
 
-       date=$(date --date=@${val} "${date_format}" 2>/dev/null)
-       [ -z "${date}" ] && date=$(date "${date_format}" 2>/dev/null)
-       [ -z "${date}" ] && date=$(date --date=@${val} 2>/dev/null)
-       [ -z "${date}" ] && date=$(date 2>/dev/null)
+       date_w=$(date --date=@${val} "${date_format}" 2>/dev/null)
```

Comment:
what does `_w` mean?

Reply:
Hi Ilya, It's the date of the warning... I was overriding the date variable before, it needs to be a different var.

Reply:
i see, was about to suggest to name it `warning_date` (or `date_warning`) 😄

Reply:
We could yes :-)

Many of these will go through proper review when they go into master, so they can/will be changed !